R/Subsets.R

```diff
@@ -782,7 +782,11 @@ LimitSubsetOperator <- R6::R6Class(
         return(private$.limitTo)
       }
       checkmate::assertCharacter(limitTo)
-      checkmate::assertChoice(limitTo, choices = c("", "firstEver", "earliestRemaining", "latestRemaining", "lastEver"))
+      checkmate::assertChoice(limitTo, choices = c("", "all", "firstEver", "earliestRemaining", "latestRemaining", "lastEver"))
```

Comment:
Is "" a choice now that we have 'all'?

Reply:
the change maintains backwards compatibility with old definitions and replaces "" with "all"